sos/report/plugins/powerpc.py

```diff
@@ -87,7 +92,8 @@ def setup(self):
                 "lsslot",
                 "amsstat"
             ])
-            self.add_service_status("hcn-init")
+            self.add_service_status(["hcn-init",
+                                     "ctrmc.service"])
```

Comment:
Minor nitpick - sos style is to indent like so:

```python
self.add_service_status([
    'hcn-init',
    'ctrmc'
])
```

Also note that you should not need to include the `.service` suffix for this call.

Reply:
Thanks for the review @TurboTurtle 

Let me update the patch.

Do you recommend checking sos changes with any python linter to avoid such mistakes in future?

Reply:
Thanks for the input @TurboTurtle